cpp/src/arrow/compute/exec/aggregate_node.cc

```diff
@@ -225,6 +236,23 @@ class ScalarAggregateNode : public ExecNode {
 
   Future<> finished() override { return finished_; }
 
+ protected:
+  std::string ToStringExtra() const override {
+    std::stringstream ss;
+    const auto input_schema = inputs_[0]->output_schema();
+    ss << "aggregates=[" << std::endl;
```

Comment:
I see this code repeated in group by node. Is there anyway we can make a function that wraps this functionality so we dont't have to update it twice if that ever becomes necessary?

Reply:
Sure, pulled out a helper function.